frontend/README.md

```diff
@@ -19,6 +19,11 @@ $ yarn generate
 
 For detailed explanation on how things work, check out the [documentation](https://nuxtjs.org).
 
+## To launch the frontend 
+
+To launch the frontend, create an .env file and point the frontend towards the back. Like this: 
```

Comment:
It might be worth mentioning that in order to create the `.env` file he should probably copy the `.env.example` file and edit it.